bower.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "colette",
-    "version": "0.29.0",
```

Comment:
We should up minor version because if colette users use navbar they must update code.

Reply:
I originally made a minor up. 0.29.0 --> 0.29.1 but maybe you meant the contrary ?
I changed it for 0.30.0